sonic_platform_base/module_base.py

```diff
@@ -541,6 +616,8 @@ def get_midplane_ip(self):
         line-card and return the midplane IP-address of the line-card.
         When called from the line-card, the module will represent the
         Supervisor and return its midplane IP-address.
+        When called from the DPU, returns the midplane IP-address of the dpu-card.
```

Comment:
@rameshraghupathy This is a module_base Class so this will not be called from DPU.

Reply:
Fixed it.